public/css/styles.css

```diff
@@ -120,6 +120,16 @@ td {
 .address + .address {
   margin-top: 1rem;
 }
+.inline-img {
```

Comment:
This name is misleading. I was accepting that it sets something like `display: inline-block`, but it changes only padding.

I would prefer if the link will look like this:
![ezgif-3-95d991cd59](https://user-images.githubusercontent.com/2865988/38016988-137b45b2-3271-11e8-9951-08dfaba56994.gif)